extension/options/src/App.js

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from "react";
+
+const defaultOptions = {
+  arrowParens: "avoid",
+  bracketSpacing: true,
+  jsxBracketSameLine: false,
+  printWidth: 80,
+  semi: true,
+  singleQuote: false,
+  tabWidth: 2,
+  trailingComma: "none",
+  useTabs: false
+};
+
+function App() {
+  const [options, setOptions] = useState();
+
+  useEffect(() => {
+    window.chrome.storage.sync.get(defaultOptions, setOptions);
```

Comment:
Do we want to scope these settings to a property (`config`, maybe?) in case we want to add other types of data in here later? That way we wouldn't have to make this code backwards compatible in the future.

Reply:
That's a good point, I wasn't thinking about this. It seems like we could store all the options under one object and manually merge them in.